### DISCUSSION TITLE
Reaching out for mentors for project.

### DISCUSSION THREAD

**Discussion by BlossomBreeze25** [openvinotoolkit/openvino#23209] [Google Summer of Code]
Respected sir / ma'am,
I am Ragini Devi Thakkar , 4th Semester BCA(AI) student, want to join your organization. i have knowledge about AI and languages like python , java . html , css . i am interested in this project (7. OpenVINO adapters - PyTorch adapter) and I would love to contribute to your organization but could you please tell me how to get started?

Hoping to hear from you soon.

Regards,

Ragini Devi Thakkar.

**Comment by adrianboguszewski** [comment]
@jiwaszki @akuporos please look at it
